Resources/Prototypes/Entities/Structures/Storage/Closets/wardrobe.yml

```diff
@@ -193,6 +193,12 @@
   parent: WardrobePrison
   name: cargo wardrobe
 
+- type: entity
+  id: WardrobeSalvage
+  parent: WardrobeMixed
+  name: salvage wardrobe
+  description: Notably not salvaged.
```

Comment:
no fill?

Reply:
Entity fill is stored in Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml per standard.